src/codesyncd/handlers/tabs_handler.ts

```diff
@@ -0,0 +1,159 @@
+/*
+Validate size of data:
+// Diff data to be sent to server
+                const diffToSend = diffHandler.createDiffToSend(fileId);
+                diffsSize += JSON.stringify(diffToSend).length;
+                // Websocket can only accept data upto 16MB, for above than that, we are reducing number of diffs per iteration to remain under limit.
+                if (diffsSize > DIFF_SIZE_LIMIT) continue;
+                validDiffs.push(diffToSend);
+*/
+
+import { glob } from "glob";
+import { ITabYML, ITabFile } from "../../interface";
+import { generateSettings } from "../../settings";
+import { readYML } from "../../utils/common";
+import { ConfigUtils } from "../../utils/config_utils";
+import { getRandomIndex, getTabsBeingProcessed } from "../utils";
+import path from "path";
```

Comment:
Kindly follow this order
1. Built in imports at top
2. 3rd party imports at second section
3. Code imports at 3rd level